### PR TITLE
Added the changes for updated omnitruck Go version

### PR DESCRIPTION
**Pull request by vviveksharma**
## Description

- Added the changes for updated omnitruck Go version
## Related Issue

-  https://chefio.atlassian.net/browse/CHEF-13568
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] If `Gemfile.lock` has changed, I have used `--conservative` to do it and included the full output in the Description above.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

## Demos:
- https://progresssoftware.sharepoint.com/:v:/s/ChefCoreC/EePqLk0hiiRHmOKhO2L4cUEBaDxYu0nO5hl52UqO8eRmlg?e=fgPL15

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19-alpine as stage1
+FROM golang:1.22-alpine as stage1
```
Comment (vipin230) [inline Dockerfile:1]:
```suggestion
FROM golang:1.22.4-alpine as stage1
```

File: Dockerfile [hunk 2]
```diff
@@ -15,7 +15,7 @@ EXPOSE 3000
 EXPOSE 3001
 EXPOSE 3002 
 
-FROM golang:1.19-alpine 
+FROM golang:1.22-alpine
```
Comment (vipin230) [inline Dockerfile:18]:
```suggestion
FROM golang:1.22.4-alpine 
```

File: Dockerfile (hunk 1 above)
Comment (vviveksharma) [reply Dockerfile]: done

File: Dockerfile (hunk 2 above)
Comment (vviveksharma) [reply Dockerfile]: Done

**Review by sandhi18 — APPROVED** [review]

**Review by RoyShravani — APPROVED** [review]

File: .expeditor/verify.pipeline.yml [hunk 3]
```diff
@@ -1,12 +1,12 @@
 steps:
-  - label: ":go: build unit tests"
-    command:
-      - scripts/install_golang.sh
-      - go version
-      - make test
-    expeditor:
-      executor:
-        docker:
+  # - label: ":go: build unit tests"
```
Comment (rishabhjhs) [inline .expeditor/verify.pipeline.yml:2]: Delete this code

**Review by ayush-billore — APPROVED** [review]

File: .expeditor/verify.pipeline.yml (hunk 3 above)
Comment (vviveksharma) [reply .expeditor/verify.pipeline.yml]: Done

**Review by rishabhjhs — APPROVED** [review]

**Review by vipin230 — APPROVED** [review]

**Pull request merged**